docs/architecture/architecture.md

```diff
@@ -609,6 +614,117 @@ rootfs by calling `mount(8)` within the counter.  If the devicemapper block devi
 is used, `/` will be mounted on `/dev/vda`.  Users can disable direct mounting
 of the underlying block device through the runtime configuration.
 
+## Kubernetes support
+[Kubernetes\*](https://github.com/kubernetes/kubernetes/) is the dominant open source
```

Comment:
> Kubernetes is the dominant open source ...

This may be true, but I wouldn't mention that for 2 reasons:

- it may not be true tomorrow :)
- it's arguably an opinion.

Reply:
Agree. Maybe s/dominant/popular/